### PR TITLE
Detect Scoop installation of composer, fix #136

### LINKED ISSUE #136: Symfony CLI cannot find composer installed by scoop
I have just installed scoop and freshly installed symfony-cli and composer. When using symfony-cli to create a new project I am given a warning that composer could not be found and terminates. Since the recommended installation method is to use scoop, symfony-cli should be able to find composer as installed by scoop.

```bash
$ symfony new my-new-project
* Creating a new Symfony project with Composer
  WARNING: Unable to find Composer, downloading one. It is recommended to install Composer yourself at https://getcomposer.org/download/
 |

  unable to find composer, get it at https://getcomposer.org/download/: unable to setup Composer

$ which symfony
/c/Users/shaun/scoop/shims/symfony

$ which composer
/c/Users/shaun/scoop/shims/composer
```

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

**Comment by tucksaun** [comment]
hey @shaunhardy could you give it a try?

File: local/php/composer_windows.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+package php
+
+import (
+	"github.com/mitchellh/go-homedir"
+	"os"
+	"path/filepath"
+)
+
+func findComposerSystemSpecific(extraBin string) string {
```
Comment (suabahasa) [inline local/php/composer_windows.go:28]: Hi @tucksaun, I have made a small adjustment, and it works.

```
func findComposerSystemSpecific(extraBin string) string {
	// Special Support for Scoop
	paths := os.Getenv("Path")
	for _, path := range filepath.SplitList(paths) {
		if path == "" || !strings.Contains(path, "scoop\\shims") {
			continue
		}
		
		pharPath := filepath.Join(filepath.Dir(path), "apps", "composer", "current", "composer.phar")
		d, err := os.Stat(pharPath)
		
		if err != nil {
			continue
		}
		if m := d.Mode(); !m.IsDir() {
			// Yep!
			return pharPath
		}
	}
}
```

**Comment by suabahasa** [comment]
Hi @tucksaun,

Any progress on this?

I'm a Windows and Scoop user as well. 

I have made a minor adjustment, and it does work properly.  Submitted a PR on https://github.com/tucksaun/symfonycli/pull/2

Could you give it a try?

**Comment by tucksaun** [comment]
I don't have a Windows computer available at the moment so it's not easy to work on this.
But I should have one available during the Christmas holidays and be able to finish this PR at this moment.

**Comment by tucksaun** [comment]
@suabahasa I just pushed an update, can you give it a go?

**Comment by suabahasa** [comment]
before (Symfony CLI version 5.4.21)

```

suabahasa ❯ symfony new test
* Creating a new Symfony project with Composer
  WARNING: Unable to find Composer, downloading one. It is recommended to install Composer yourself at https://getcomposer.org/download/
  (running C:\Users\suabahasa\.symfony5\composer\composer.phar create-project symfony/skeleton C:\Users\suabahasa\Downloads\Compressed\symfony\new\test  --no-interaction)

```
<hr />

after (#166)

```

suabahasa ❯ .\symfony-cli.exe new test
* Creating a new Symfony project with Composer
  (running C:\Users\suabahasa\scoop\apps\composer\current\composer.phar create-project symfony/skeleton C:\Users\suabahasa\Downloads\Compressed\symfony\new\test  --no-interaction)

```

Hi @tucksaun,
It now detects the composer installed by Scoop.

**Pull request merged**